docs/content/preview/migrate/airgapped.md

```diff
@@ -0,0 +1,46 @@
+<!--
++++
+private=true
++++
+-->
+
+You can perform an airgapped installation on RHEL 7/8, CentOS 7/8, and Ubuntu OS.
+
+For airgapped installations, you need to download the tarball containing all the rpm files that are necessary for installing and running `yb-voyager` on a machine with an internet connection. Transfer the downloaded files to your airgapped machine and proceed with the installation using the following steps:
+
+1. Download the tarball containing all the rpm files on a machine with internet connection using the following command:
+
+    ```sh
+    // For RHEL 7 or CentOS 7
+    wget https://downloads.yugabyte.com/repos/airgapped/airgapped-rhel7.tar.gz
+    ```
+
+    ```sh
+    // For RHEL 8 or CentOS 8
+    wget https://downloads.yugabyte.com/repos/airgapped/airgapped-rhel8.tar.gz
+    ```
+
+    ```sh
+    // For Ubuntu
+    wget https://downloads.yugabyte.com/repos/airgapped/airgapped_ubuntu.tar.gz
+    ```
```

Comment:
We should not give the installer bundles to the user. Instead, give them the list of steps to prepare the installer bundle. @ShivanshGahlot 

It is important that, the RHEL/CentOS machine used to prepare the installer bundle and the machine used to install voyager, MUST have exact same existing packages on them.